src/attribute.rs

```diff
@@ -53,7 +53,7 @@ impl<'a> Attribute<'a> {
                 nc_inq_attlen(
                     self.ncid,
                     self.varid,
-                    self.name.as_ptr() as *const _,
+                    self.name.as_ptr().cast(),
```

Comment:
TIL :smile:

Reply:
`as _` casts have gotten very frowned upon in later editions